tests/Mono.Android-Tests/Java.Interop/JnienvTest.cs

```diff
@@ -32,11 +32,22 @@ public void TestMyPaintColor ()
 			}
 		}
 
+		[DllImport ("reuse-threads")]
+		static extern int rt_register_type_on_new_thread (string java_type_namem, IntPtr class_loader);
+
 		delegate void CB (IntPtr jnienv, IntPtr java_instance);
 
 		[DllImport ("reuse-threads")]
 		static extern int rt_invoke_callback_on_new_thread (CB cb);
 
+		[Test]
+		public void RegisterTypeOnNewThread ()
+		{
+			Java.Lang.JavaSystem.LoadLibrary ("reuse-threads");
+			int ret = rt_register_type_on_new_thread ("from/NewThread", Application.Context.ClassLoader.Handle);
```

Comment:
Shouldn't this be:

```c
rt_register_type_on_new_thread ("from.NewThread", Application.Context.ClassLoader.Handle);
```

because `rt_register_type_on_new_thread()` uses `ClassLoader.loadClass()`?